packages/documentation/docs/user-guide/installation/installing-a-gateway/intro.md

```
@@ -1,25 +1,41 @@
 ---
 sidebar_label: Introduction
-sidebar_position: 1
+sidebar_position: 10
 ---
 # Introduction: Installing a Gateway
 
 #### Prerequisites
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)